doc/guides/tfm/requirements.rst

```diff
@@ -0,0 +1,56 @@
+TF-M Requirements
+#################
+
+The following boards can be used with TF-M at present:
```

Comment:
Dangerous; this is going to go stale quickly.

Maybe say "The following example boards [...]", and also add docs for how users can check if their board is supported? This would also ease the lives of module vendors with out of tree boards.

Reply:
Let me know if the current update is acceptable, thanks.

Reply:
Thanks for the update!

`git grep CONFIG_TRUSTED_EXECUTION_NONSECURE=y boards` would work on Windows too (I don't think `-i` is needed; is it?), but that would still only catch in-tree boards. 

(I care about this in particular because we have an out of tree nRF9160 board in our downstream SDK at Nordic, so I'm one of those "module vendors with out of tree boards" :wink: .)

What about something like this?

```
You can run ``west boards -n _ns`` to search for non-secure variants
of different board targets. To make sure TF-M is supported for a board
in its output, check that :kconfig:`CONFIG_TRUSTED_EXECUTION_NONSECURE`
is set to ``y`` in that board's default configuration.
```

Reply:
Actually, `west boards -n _ns$` would be better.

Reply:
I was worried that the _ns suffix might not be good on its own since there are others TEEs besides TFM, which is why I added a search for the config flag as well.

Reply:
I ended up pushing this, which I think is clear enough for people:

```
You can run ``west boards -n _ns$`` to search for non-secure variants
of different board targets. To make sure TF-M is supported for a board
in its output, check that :kconfig:`CONFIG_TRUSTED_EXECUTION_NONSECURE`
is set to ``y`` in that board's default configuration.
```